impressum/README.md

```diff
@@ -7,7 +7,7 @@ Impressum Addon for Friendica
 
 About
```

Comment:
Could you please correct the format of this file to the usual "one-sentence-per-line" format of documentation files. It is markdown, so should be rendered correctly by repository hosting and in the admin panel.